SWLOR.Game.Server/Feature/PerkDefinition/ForcePerkDefinition.cs

```diff
@@ -680,5 +683,86 @@ private void ForceInspiration()
                 .RequirementCharacterType(CharacterType.ForceSensitive)
                 .GrantsFeat(FeatType.ForceInspiration3);
         }
+
+
+        private void ForceDeath()
+        {
+            _builder.Create(PerkCategoryType.ForceDark, PerkType.ForceDeath)
+                .Name("Force Death")
+
+                .AddPerkLevel()
+                .Description("Deals 30 damage, and gives yourself 40 temporary hp")
+                .Price(3)
+                .RequirementSkill(SkillType.Force, 25)
+                .RequirementCharacterType(CharacterType.ForceSensitive)
+                .RequirementCannotHavePerk(PerkType.Benevolence)
+                .GrantsFeat(FeatType.ForceDeath1)
+
+                .AddPerkLevel()
+                .Description("Deals 45 damage, and gives yourself 55 temporary hp")
+                .Price(3)
+                .RequirementSkill(SkillType.Force, 35)
+                .RequirementCharacterType(CharacterType.ForceSensitive)
+                .RequirementCannotHavePerk(PerkType.Benevolence)
+                .GrantsFeat(FeatType.ForceDeath2)
+
+                .AddPerkLevel()
+                .Description("Deals 60 damage, and gives yourself 70 temporary hp")
+                .Price(3)
+                .RequirementSkill(SkillType.Force, 45)
+                .RequirementCharacterType(CharacterType.ForceSensitive)
+                .RequirementCannotHavePerk(PerkType.Benevolence)
+                .GrantsFeat(FeatType.ForceDeath3);
+
+                
+                
+
+
+
+        }
+
+        
+
+         private void Tutaminis()
+        {
+            _builder.Create(PerkCategoryType.ForceUniversal, PerkType.Tutaminis)
+                .Name("Tutaminis")
+
+                .AddPerkLevel()
+                .Description("Increase your defense by 10")
```

Comment:
I'm still unclear as to whether this is implemented but please specify whether it is Physical, Force, or a specific element's defense in the description.